src/webview_provider/docsEditPanel.ts

```diff
@@ -417,7 +417,10 @@ export class DocsEditViewPanel implements WebviewViewProvider {
       }
 
       // new column, save the name by checking the config
-      return { ...c, name: getColumnNameByCase(c.name) };
+      return {
```

Comment:
This is while syncing with db and we should send name as the case received from db to webview, so while saving we will be able to identify right case and quote attribute